src/components/cart-products.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { useCartContext } from "../context/cart-context";
+import { useState } from "react";
+
+function Cartproducts(props) {
+  const { cartState, cartDispatch } = useCartContext();
+
+  console.log(cartState.cartItem);
```

Comment:
Do, remove this log while merging.

Reply:
Removed console.log, Thanks for pointing out